app/views/content/life-as-a-teacher/age-groups-and-specialisms/age-groups-you-could-teach.md

```diff
@@ -6,6 +6,16 @@ description: |-
 cta_mailinglist:
   mailinglist:
     text: Get free guidance and support on getting into teaching, including learning more about the age groups you could teach.
+cta_arrow_link:
+  subjects:
+    link_target: "/life-as-a-teacher/explore-subjects"
+    link_text: "Explore the subjects you could teach at secondary school level"
+  schoolexperience:
+    link_target: "/train-to-be-a-teacher/get-school-experience"
+    link_text: "Find out more about opportunities to get school experience in England"
+  send:
+    link_target: "/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs"
+    link_text: "Learn more about the routes to teach pupils with special educational needs and disabilities (SEND)"
```

Comment:
```suggestion
    link_text: "Learn more about teaching pupils with special educational needs and disabilities (SEND)"
```
or `training to teach` rather than `the routes to teach`?